src/test/java/com/github/mauricioaniche/ck/FieldsTest.java

```diff
@@ -9,48 +9,61 @@
 public class FieldsTest extends BaseTest {
```

Comment:
It seems all tabs/spaces were changed, so the PR is longer than it needs to be. Not a problem for this one, but for the next ones, try to avoid it!